pkgs/development/libraries/geos/default.nix

```diff
@@ -21,8 +21,8 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
 
   meta = with lib; {
-    description = "C++ port of the Java Topology Suite (JTS)";
-    homepage = "https://trac.osgeo.org/geos";
+    description = "C/C++ library for computational geometry with a focus on algorithms used in geographic information systems (GIS) software";
+    homepage = "https://libgeos.org";
```

Comment:
Not wanting to abuse your benevolence, but can you add `mainProgram = "<the executable name>";`?

Reply:
This package is a library. However, there are two "programs" (in `bin` there is [`geosop`](https://libgeos.org/usage/tools/#geosop) and `geos-config`). Neither of these are the main aspects of the package, would they be listed with mainProgram?

Reply:
> geosop is a CLI (command-line interface) for GEOS.

It looks a good candidate if you ask me.

Well, `nix run` tries to run `meta.mainProgram`, and it can be useful for a smoke test. 
Even if it is not a main aspect of package, this is still nice to set.